core/src/main/java/org/apache/gravitino/auxiliary/AuxiliaryServiceManager.java

```diff
@@ -234,18 +234,17 @@ static Map<String, String> extractAuxiliaryServiceConfigs(Config config) {
             .getOrDefault(AUX_SERVICE_NAMES, "");
     Map<String, String> serviceConfigs = new HashMap<>();
     serviceConfigs.put(AUX_SERVICE_NAMES, auxServiceNames);
-    config
-        .getAllConfig()
+    config.getAllConfig().entrySet().stream()
+        .filter(entry -> !entry.getKey().equals("gravitino.auxService.names"))
```

Comment:
could you replace `gravitino.auxService.names` with `GRAVITINO_AUX_SERVICE_PREFIX + AUX_SERVICE_NAMES`?

Reply:
Sure 😺